packages/form-builder/src/blocks.json

```diff
@@ -0,0 +1,96 @@
+[
+    {
+        "clientId": "8371d4c7-0e8d-4aff-a1a1-b4520f008132",
```

Comment:
@kjohnson Out of curiosity, what's the "clientId" all about?

Reply:
The `clientId` is a required property for the Block List, used as a unique identifier. I add it here simply to as to avoid generating it after localizing the data. The specific value is not important to us, but I sought to avoid the extra processing at this time.

As a note, the `parse` method probably re-generates this when converting the HTML comment markup to JavaScript, but I added it here for simplicity.